contracts/contracts/coordination/TACoChildApplication.sol

```diff
@@ -112,10 +113,9 @@ contract TACoChildApplication is AccessControl, ITACoRootToChild, ITACoChildAppl
         require(msg.sender == coordinator, "Only Coordinator allowed to confirm operator");
         address stakingProvider = stakingProviderFromOperator[_operator];
         StakingProviderInfo storage info = stakingProviderInfo[stakingProvider];
-        if (info.operatorConfirmed) {
-            return;
-        }
         require(info.authorized > 0, "No stake associated with the operator");
+        // TODO maybe allow second confirmation, just do not send root call
+        require(!info.operatorConfirmed, "Can't confirm same operator twice");
```

Comment:
IMHO I'd just throw, but I have a bias for throwing

Reply:
Throw will allow to use same tx again, and I don't think it would be good. If tx succeeded then you can't use it twice in a bridge